app/Services/LanguagePackageServiceProvider.js

```diff
@@ -47,8 +47,32 @@ async function destroyLanguagePackage(userId, languagePackageId) {
   await languagePackage.destroy();
 }
 
+async function updateLanguagePackage(
+  { name, foreignWordLanguage, translatedWordLanguage, vocabsPerDay, rightWords },
+  userId,
+  languagePackageId
+) {
+  const languagePackage = await LanguagePackage.findOne({
+    where: {
+      id: languagePackageId,
+      userId,
+    },
+  });
+
+  languagePackage.name = name;
+  languagePackage.foreignWordLanguage = foreignWordLanguage;
+  languagePackage.translatedWordLanguage = translatedWordLanguage;
+  languagePackage.vocabsPerDay = vocabsPerDay;
+  languagePackage.rightWords = rightWords;
```

Comment:
For that we can use the update [method](http://sequelize.org/master/class/lib/model.js~Model.html#static-method-update) from sequelize.
```js
await languagePackage.update(data, {
  fields: ["name", "foreignWordLanguage", "translatedWordLanguage", "vocabsPerDay", "rightWords"]
})
```
The fields is to prevent the user from updating the `userId` or other columns.

Reply:
I will add fields to other functions too